vendor.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/miekg/dns v1.1.43
 	github.com/mistifyio/go-zfs/v3 v3.0.1
 	github.com/mitchellh/copystructure v1.2.0
-	github.com/moby/buildkit v0.12.5
+	github.com/moby/buildkit v0.12.6-0.20250422135724-b10aeed77fd8
```

Comment:
I think the update in buildkit only impacts CI, not the vendored code https://github.com/moby/buildkit/compare/v0.12.5...b10aeed77fd8

So as alternative, it might be cleaner to override the version used for running the tests; that way the module used sticks to a tagged version, and only CI is updated with the changes needed to run it; similar to what we did in;

- https://github.com/moby/moby/pull/44701